python-backend/app/commands.py

```diff
@@ -111,13 +111,15 @@ def _create_data(num):
 
                 first_name = names.get_first_name()
                 last_name = names.get_last_name()
-                email = first_name.lower() + '.' + last_name.lower() + '@' + last_name.lower() + '.com'
-                party = Party.create(last_name,
-                                     email,
-                                     '123-123-1234',
-                                     'PER',
-                                     DUMMY_USER_KWARGS,
-                                     first_name=first_name)
+                email = first_name.lower() + '.' + last_name.lower() + '@' + last_name.lower(
+                ) + '.com'
```

Comment:
Do we want string interpolation here?

Reply:
I have no Idea, this was a formatter change.

Reply:
`f'{first_name.lower()}.{last_name.lower()}@{last_name.lower()}.com'` 

up to you.

Reply:
Changed